src/java_bytecode/java_local_variable_table.cpp

```diff
@@ -186,9 +187,10 @@ static bool is_store_to_slot(
   else
   {
     // Store shorthands, like "store_0", "store_1"
-    assert(prevstatement[6]=='_' && prevstatement.size()==8);
+    INVARIANT(prevstatement[6]=='_' && prevstatement.size()==8,
+      "expected store instruction looks like store_0, store_1...");
     storeslot=prevstatement[7];
-    assert(isdigit(storeslot[0]));
+    INVARIANT(isdigit(storeslot[0]), "digit expected");
```

Comment:
Description doesn't add anything on top of the code. How about "`store_ instructions should end in a digit`"